storage/container_storage_interface/persistent-storage-csi-azure-file.adoc

```diff
@@ -9,21 +9,21 @@ toc::[]
 
 {product-title} is capable of provisioning persistent volumes (PVs) by using the Container Storage Interface (CSI) driver for Microsoft Azure File Storage.
 
-:FeatureName: Azure File CSI Driver Operator
-include::snippets/technology-preview.adoc[leveloffset=+1]
-
 Familiarity with xref:../../storage/understanding-persistent-storage.adoc#understanding-persistent-storage[persistent storage] and xref:../../storage/container_storage_interface/persistent-storage-csi.adoc#persistent-storage-csi[configuring CSI volumes] is recommended when working with a CSI Operator and driver.
 
-To create CSI-provisioned PVs that mount to Azure File storage assets with this feature enabled, {product-title} installs the Azure File CSI Driver Operator and the Azure File CSI driver by default in the `openshift-cluster-csi-drivers` namespace.
+To create CSI-provisioned PVs that mount to Azure File storage assets, {product-title} installs the Azure File CSI Driver Operator and the Azure File CSI driver by default in the `openshift-cluster-csi-drivers` namespace.
 
-* The _Azure File CSI Driver Operator_, after being enabled, provides a storage class that is named `azurefile-csi` that you can use to create persistent volume claims (PVCs). The Azure File CSI Driver Operator supports dynamic volume provisioning by allowing storage volumes to be created on-demand, eliminating the need for cluster administrators to pre-provision storage.
+* The _Azure File CSI Driver Operator_ provides a storage class that is named `azurefile-csi` that you can use to create persistent volume claims (PVCs). The Azure File CSI Driver Operator supports dynamic volume provisioning by allowing storage volumes to be created on-demand, eliminating the need for cluster administrators to pre-provision storage.
 
 * The _Azure File CSI driver_ enables you to create and mount Azure File PVs.
 
-include::modules/persistent-storage-csi-about.adoc[leveloffset=+1]
+Azure File CSI Driver Operator does not support:
```

Comment:
I thought we said we don't support snapshots as well. @lpettyjo @bertinatto

Reply:
Non-support for snapshots is documented in the table shown in modules/persistent-storage-csi-drivers-supported.adoc.

Reply:
thanks, i think it would be best to gather all unsupported features here as well

Reply:
I've added a link to the table.